xclim/indices/_multivariate.py

```diff
@@ -567,8 +567,8 @@ def days_over_precip_thresh(pr, per, thresh="1 mm/day", freq="YS"):
     ----------
     pr : xarray.DataArray
       Mean daily precipitation flux [Kg m-2 s-1] or [mm/day]
-    per : xarray.DataArray
-      Daily percentile of wet day precipitation flux [Kg m-2 s-1] or [mm/day]
+    per : xarray.DataArray or string (scalar with units)
```

Comment:
I don't think this will work with IDEs. From my understanding, this field acts as a call signature hint, the same as what we might find in `def function(a: str = 'foo', b: int or float = 123.321, c: xarray.DataSeries = ds)`.
 
Outside of types joined with `or` or with `[ ]`, the only other thing allowed for this type of docstring is the `, optional` indicator that references the `typing` class of `Optional` (`def func(a: Optional[str] = None`). 

I'd suggest adding that the units are needed in the docstring and just raise a verbose ValueError if there aren't any units. 
```suggestion
    per : xarray.DataArray or str
```